pkg/api/kyverno/v1/types.go

```diff
@@ -56,13 +56,13 @@ type PolicyViolationList struct {
 type Policy struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
-	Spec              Spec         `json:"spec"`
+	Spec              Spec         `json:"spec,omitempty"`
```

Comment:
Do we need all the types to set as `omitempty` ?

Reply:
Removed.